BridgeAppSDK/SBAUserWrapper+Bridge.swift

```diff
@@ -232,7 +232,7 @@ public extension SBAUserWrapper {
      @param completion  Completion handler
      */
     public func verifyRegistration(_ completion: ((Error?) -> Void)?) {
-        guard let username = self.email?(forAuthManager: nil), let password = self.password?(forAuthManager: nil) else {
+        guard let username = self.email?(forAuthManager: nil), let password = self.password(forAuthManager: nil) else {
```

Comment:
yes, I left it optional. I can't imagine not wanting it, an argument could be easily made for requiring it, but it's always been optional and nothing about its actual technical necessity has changed.